src/components/Layout/CardVideo/CardVideo.component.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+
+
+const CardContainer = styled.div`
+    width: 350px;
+    border-radius: 5px;
+    border: 1px solid #ddd;
+    margin: 10px;
+    overflow: hidden;
+    padding: 15px;
+`;
+
+const CardImg = styled.img`
+    width: 100%;
+    height: 140px;
+`;
+
+
+function CardVideo(props){
```

Comment:
You can extract the props using destructuring, so it is cleaner in your component, like so:

```javascript
function CardVideo({ item }) {
```

Check the comment about the extra `{}` cause I think it wasn't intended.

Reply:
Now that you are using props, it could be useful to put in place a set of rules for those props, so that the component expects exactly what you meant. Check the documentation for these and let me know if you have any questions:

https://reactjs.org/docs/typechecking-with-proptypes.html